packages/ilios-common/addon/routes/course-visualize-instructor.js

```diff
@@ -15,7 +15,7 @@ export default class CourseVisualizeInstructorRoute extends Route {
   }
 
   async afterModel({ course }) {
-    const sessions = (await course.sessions).slice();
+    const sessions = await course.sessions;
     return await all([course.school, map(sessions, (s) => s.sessionType)]);
```

Comment:
Interesting thing here. It looks like `RSVP.map` will resolve a promise that is passed as the first argument. Really for discussion, but would it be cleaner to do this without the interim variable as just `map(course.sessions, fn)`?

Reply:
Seems like a cromulent change. Probably need to do this across the site, but I'm checking to see if the change kills any tests now.